applications/SwimmingDEMApplication/tests/InterpolationTestFactory.py

```diff
@@ -5,7 +5,7 @@
 
 # Import KratosUnittest
 import KratosMultiphysics.KratosUnittest as KratosUnittest
-from tests_python_scripts.interpolation_scripts.interpolation_test_analysis import InterpolationTestAnalysis
+from KratosMultiphysics.SwimmingDEMApplication.tests_python_scripts.interpolation_scripts.interpolation_test_analysis import InterpolationTestAnalysis
```

Comment:
@philbucher : this was the problematic line. We tried many different combinations here, in order to avoid changing the core. It is clear that we are inside the 'tests' folder and we want to load a python module (which might be independent from the application, if you want) inside of 'tests' as well.
I will try a bit more, but if you have any proposal to fix this line, it is welcome.

Reply:
Forget about it, I solved it. I am pushing in a minute.

Reply:
hm strange, I would have said that the original version should also work :confused: 

Maybe the `__init__.py`s have an influence, but I am not sure, probably not ...

Which script do I have to run to try? The regular testing-script?
Then I could also try myself

Reply:
cool!
(didn't see your comment)